### ISSUE TITLE
Bug: conflicting version requirements of zone.js

### ISSUE THREAD

**Issue by JensSteckhan** [SAP/fundamental-ngx#3702]
#### Is this a bug, enhancement, or feature request?
This is a bug.

#### Briefly describe your proposal.
I tried to update my project to fundamental-ngx 0.23, it was already running on angular 10 (10.0.5 to be exact).
So I updated my package json to 10.1.5 and ran into a conflicting specification of the required zone.js version.

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
fundamental 0.23, angular 10.1.5

#### If this is a bug, please provide steps for reproducing it.
There are conflicting requirements for the zone.js library.
If I configure package.json as `"zone.js": "~0.11.2"`, I get the following error:
`@angular/core@10.1.5 requires a peer of zone.js@~0.10.3 but none is installed. You must install peer dependencies yourself.`

If I configure `"zone.js": "~0.10.3"`, then I get:
`@fundamental-ngx/core@0.23.0 requires a peer of zone.js@0.11.2 but none is installed. You must install peer dependencies yourself.`

If I try to use a lower version of angular (e.g. 10.1.4), I get this error:
`@fundamental-ngx/core@0.23.0 requires a peer of @angular/core@~10.1.5 but none is installed. You must install peer dependencies yourself.
`
Moving up to angular 10.1.6 doesn't fix the problem and fundamental-ngx does not support any newer versions (angular 10.2.0) either.

In other words: **fundamental-ngx/core makes conflicting demands: it specifies angular 10.1.5., which needs zone.js 0.10.3, but at the same time it specifies zone.js 0.11.2.**

**Comment by JensSteckhan** [comment]
The problem is still there with fundamental 0.24.1:

@fundamental-ngx/core@0.24.1 requires a peer of zone.js@0.11.2 
but
@angular/core@10.2.3 requires a peer of zone.js@~0.10.3

**Comment by JensSteckhan** [comment]
#4467 seems to be a duplicate

**Comment by InnaAtanasova** [comment]
Hello @JensSteckhan, did you migrate to Angular 11? I created a new Angular project and added @fundamental-ngx/core@0.24.1. I did not run into this problem.

**Comment by JensSteckhan** [comment]
Hi @InnaAtanasova! 
I had tried to migrate to Angular 11 and the latest fundamental-ngx release, but the same problem occurred there as well.
Thank you for looking into it!

**Comment by JensSteckhan** [comment]
Hi @InnaAtanasova! Any news?

**Comment by dimamarksman** [comment]
Hi @JensSteckhan I've tried to run @angular 10.2.5 with @fundamental-ngx 0.24.1 and I don't see the error. 
There is only a warning
![image](https://user-images.githubusercontent.com/3263744/118283375-25ee7780-b484-11eb-9f7f-1934847d19b8.png)

**Comment by JensSteckhan** [comment]
Hi,

it is simple to reproduce: include both angular/core 10.2.5 and fundamental-ngx/core 0.26.1 in the package json of a project.

If you include zone 0.11.3 (as required by fundamental-ngx/core 0.26.1), you will get this error:
![image](https://user-images.githubusercontent.com/9308805/119613611-92129900-bdfd-11eb-8b52-3f15d83c18c1.png)

If you include zone (as required by angular/core 10.2.5), you will get this:
![image](https://user-images.githubusercontent.com/9308805/119613880-e7e74100-bdfd-11eb-9919-4a8a4b1a9e10.png)

If you simply leave out zone.js in package.json, npm i will complain that it is missing:
![image](https://user-images.githubusercontent.com/9308805/119614087-2846bf00-bdfe-11eb-8fd1-274d18f668c9.png)

In other words: it is impossible to use fundamental in your project and do a regular npm i.

**Comment by droshev** [comment]
@jenshuesken Can you try with npm v6?
